MahApps.Metro/Controls/Helper/VirtualKeyboardHelper.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Windows;
+using System;
+using System.Linq;
+using System.Management;
+
+namespace MahApps.Metro.Controls.Helper
+{
+    public class VirtualKeyboardHelper
+    {
+        private const uint SC_CLOSE = 61536;
+        private const uint WM_SYSCOMMAND = 274;
+        private static DateTime lastCloseRequest;
+
+        private static readonly Lazy<ProcessStartInfo> StartInfo = new Lazy<ProcessStartInfo>(() =>
+        {
+            ProcessStartInfo startInfo;
+            if (Environment.Is64BitOperatingSystem)
+            {
+                startInfo = new ProcessStartInfo(@"C:\Program Files\Common Files\Microsoft Shared\ink\TabTip.exe");
+            }
+            else
+            {
+                startInfo = new ProcessStartInfo(@"C:\Program Files (x86)\Common Files\Microsoft Shared\ink\TabTip32.exe");
```

Comment:
@dgadelha do you have a x86 OS? can you check whether the file TabTip32.exe exists?